### DISCUSSION TITLE
2023년 07월 20일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#69] [General]
# 2023년 07월 20일

LOG 발목이 나은 줄 알았는데 안 나았다. 역시 병원에 갈 걸 그랬나 ㅋㅋ 데모데이 전 날... 처음으로 S-HOOK 데모를 봤는데 간지 ㄷㄷ 프론트가 열심히 일해준 덕분에 무사히 데모를 할 수 있을 거 같다. 배운 것 sudo nginx -t  …

https://vero.wiki/2023-07-20/

**Comment by somsom13** [comment]
멋지네요 베로위키 ㄷㄷ
아프지마세요

**Reply by Cyma-s** [reply]
이제 멀쩡합니다 ^____^

**Comment by hong-sile** [comment]
vim에서 마우스 커서를 왜 쓰죠? hjkl web 로 이동하시죠

**Reply by Cyma-s** [reply]
진짜 머임 그건 ㅋㅋㅋㅋㅋㅋㅋ
